src/components/Settings.vue

```diff
@@ -321,12 +321,20 @@
                         ></v-switch>
                       </v-flex>
                       <v-flex xs12 v-if="gateway.hassDiscovery">
-                        <v-text-field
-                          v-model="gateway.entityTemplate"
-                          label="Entity name template"
-                          persistent-hint
-                          hint="Template which generates entity names<br>1"
-                        ></v-text-field>
+                          <v-text-field
+                            v-model="gateway.entityTemplate"
+                            label="Entity name template"
+                            persistent-hint
+                            hint="Template which generates entity names"
+                          ></v-text-field>
+                          <v-tooltip bottom>
```

Comment:
@billiaz tooltip doesn't work in that way :) Check it here: https://vuetifyjs.com/en/components/tooltips/